xbmc/cores/VideoPlayer/VideoPlayerAudio.cpp

```diff
@@ -576,6 +576,7 @@ void CVideoPlayerAudio::Process()
         msg.player = VideoPlayer_AUDIO;
         msg.cachetotal = cachetotal;
         msg.cachetime = cachetime;
+        msg.next_pts = m_messageQueue.GetTimeBack();
         msg.timestamp = m_audioClock;
```

Comment:
I'll try to describe my problem in PTS
- Video / audio starts at 0
- audio changes at 20010666.6667
- video changes at 20103416.6667

If VideoPlayer:HandlePlaySpeed is called, we have:

VideoPlayer::Sync - Audio - pts: 20053333.333333, cache: 566260.229148, totalcache: 579999.983311, nextpts: 20010666.666667

This leads to a clock value : 20053333.333333 - 566260.229148 and this is 1/2 second to less.
Question is: why is the audio cache: 566260.229148? There was an audio Streamchange .1 secs before!

Maybe the audio cache is not resetted on streamchange? Any idea?